lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1075,6 +1075,7 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
   // PE/COFF cannot deal with the cross-module reference to the metadata parent
   // (e.g. NativeObject).  Force the lazy initialization of the VWT always.
   Opts.LazyInitializeClassMetadata = Triple.isOSBinFormatCOFF();
+  Opts.LazyInitializeProtocolConformances = Triple.isOSBinFormatCOFF();
```

Comment:
Please add a comment explaining why this is necessary.

Reply:
Sure, done.